src/storage/secondary/HttpStorage.cpp

```diff
@@ -132,7 +155,10 @@ HttpStorage::HttpStorage(const Url& url, const AttributeMap& attributes)
     m_http_client(make_client(url))
 {
   m_http_client->set_default_headers(
-    {{"User-Agent", FMT("{}/{}", CCACHE_NAME, CCACHE_VERSION)}});
+    {// explicit setting of the Host header is required due to IPv6 address
+     // handling issues in httplib
```

Comment:
Is this something we should report to cpp-httplib?

Reply:
I mentioned it on the closed PR but opened a new issue: yhirose/cpp-httplib#998